core/src/main/java/org/apache/calcite/sql2rel/SqlToRelConverter.java

```diff
@@ -6374,10 +6390,13 @@ private static class SubQuery {
     final SqlNode node;
     final RelOptUtil.Logic logic;
     @Nullable RexNode expr;
+    final SqlImplementor.@Nullable Clause parentClause;
```

Comment:
How about just name it `clause` instead of `parentClause`? ('parent' makes think that it's something like 'parent scope')

Reply:
Fixed.